visual-inspection/inspection-frontend/src/components/InspectImage.vue

```diff
@@ -1,9 +1,19 @@
 <template>
   <div class="inspector">
-    <MultiBounce v-if="!prediction"
-                 v-bind:numberOfDots="3"/>
-    <p v-show="prediction && currentPrediction">„Das ist {{ prediction }}“</p>
-  </div>
+    <MultiBounce v-if="topPredictions.length===0"
+                 v-bind:numberOfDots="3"/>     
+        <span class="prediction-result" v-if="currentPrediction && topPredictions[0][1] > MinAccuracy"> 
+            Bestimmung:
+            <br />
+            <p>Die KI ist sich zu {{topPredictions[0][1]}}% sicher, dass es sich um {{topPredictions[0][0]}} handelt.</p>
+            <div v-for="(item, index) in topPredictions" :key="index"> 
+                <a v-show="index === 1">Alternativ könnte das auch </a>
+                <a v-show="index > 0">{{item[0]}} ({{item[1]}}%)</a>
+                <a v-show="index > 0 && index < topPredictions.length-1"> oder </a>
+                <a v-show="index > 0 && index === topPredictions.length-1"> sein.</a>
+            </div>
```

Comment:
I would like to understand what this is supposed to accomplish 😃 This doesn't seem to be functional (it is never shown what the prediction actually is?) and seems overly complicated.